runtime/js/27_websocket.js

```diff
@@ -265,6 +267,13 @@
           event.target = this;
           this.dispatchEvent(event);
 
+          this.#eventLoop();
+        } else if (message.type === "ping") {
```

Comment:
This is ok for a first pass, but I think it makes sense to move this logic to Rust.

Reply:
I do plan a somewhat rewrite of websocket (mostly JS things) together with moving it to its own cratet